_posts/2025-02-11-5.11.0-released.md

```diff
@@ -0,0 +1,81 @@
+
+### Table of Contents
+
+- [Compatibility Changes](#compatibility-changes)
+- [In-Game Settings Menu](#in-game-settings-menu)
+- [Touchscreen Layout Editor](#touchscreen-layout-editor)
+- [More Server List Info](#more-server-list-info)
+- [Default Font Overrides](#default-font-overrides)
+- [Other Highlights](#other-highlights)
+- [Onwards and Upwards](#onwards-and-upwards)
+
+## Compatibility Changes
+
+There are a few changes which may (or may not) affect you, so we'll get them out of the way first:
+
+* **OpenGL 2.0** is now the minimum OpenGL version required by Luanti ([here's why](https://github.com/luanti-org/luanti/issues/15370)).
+* **BMP image support has been dropped** as it was deprecated in 2023.
+* A long-standing bug with bone rotations was fixed, which [may break some mods](https://github.com/luanti-org/luanti/issues/15692) that had worked around the issue. In particular, mods overriding bones with 180° rotations (like [playeranim](https://content.luanti.org/packages/Rui/playeranim/)) may have issues until fixed. Check your mods, devs!
+
+## In-Game Settings Menu
+
+The settings menu can now be accessed from the in-game pause menu (very convenient!). Keep in mind that not every setting supports live changes, so you may still need to reconnect after making certain alterations.
+
+{% include figure.html src="/static/blog/5.11.0/ingame_settings_menu.webp" caption="The in-game settings menu" %}
+
+## Touchscreen Layout Editor
+
+For those of you on smartphones or some other touchscreen device ([smartwatch](https://www.youtube.com/watch?v=6MItj9AJbP4), anybody?), Luanti now includes a layout editor for your on-screen controls. Select `Touchscreen Layout` in the pause menu and organize the buttons to your heart's content!
+
+<figure class="figure image">
+    <video autoplay controls loop>
+        <source src="/static/blog/5.11.0/touchscreen_editor.webm" type="video/webm" />
+    </video>
+	<figcaption>The touchscreen editor in action</figcaption>
+</figure>
+
+## More Server List Info
+
+Have you ever wanted to check server info _before_ you connect? Until now we had to use the [online server list](https://servers.luanti.org/), but Luanti has introduced 4 new buttons allowing you to open the server's website, view its mod list, see which players are online, and manage your favorites.
+
+{% include figure.html src="/static/blog/5.11.0/serverlist_buttons.webp" caption="Server list info buttons" %}
+
+## Default Font Overrides
+
+Luanti's fonts are starting to get boring after more than a decade, so mods can now override the defaults! Place TTF or WOFF2 fonts in your mod's `fonts` directory for each style you want to override (check out [the documentation](https://github.com/luanti-org/luanti/blob/5.11.0/doc/lua_api.md#custom-fonts) for more details). This is just the first step towards better font control, so keep an eye out for future font features.
```

Comment:
```suggestion
Luanti's fonts are starting to get boring after more than a decade, so mods can now override the defaults! Place TTF or WOFF fonts in your mod's `fonts` directory for each style you want to override (check out [the documentation](https://github.com/luanti-org/luanti/blob/5.11.0/doc/lua_api.md#custom-fonts) for more details). This is just the first step towards better font control, so keep an eye out for future font features.
```

(it was WOFF2 initially, but we ended up with WOFF due to concerns over availability of the decompression WOFF2 depends on; sorry if I forgot to update the PR)